exc2/src/graphics.py

```diff
@@ -0,0 +1,97 @@
+"""
+Author: Gustav Kjellberg
+Email: kjellberg.p.gustav@gmail.com
+Licence: MIT
+Creation data: 22 Sept 2020
+Last modified: 24 Sept 2020
+"""
+
+import math
+import os
+import sys
+
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib import animation
+import tikzplotlib
+
+
+class Graphics():
+    """[summary]
+        A class for visualizing two-dim data
+    """    
+    def __init__(self):
+        """[summary]
+            Class initializer
+        """        
```

Comment:
```suggestion
        """Class initializer
        """        
```

Small detail: You don't need to add the word summary in the docstring :slightly_smiling_face: